src/main/java/net/mcreator/blockly/BlocklyBlockUtil.java

```diff
@@ -114,7 +115,8 @@ public static List<Element> getBlockProcedureStartingWithNext(Element start_bloc
 	 * @return Block's render number
 	 */
 	public static Color getBlockColorFromHUE(int hue) {
-		return Color.getHSBColor(hue / 360f, 0.37f, 0.6f);
+		return Color.getHSBColor(hue / 360f, PreferencesManager.PREFERENCES.blockly.colorSaturation.get() / 100f,
+				PreferencesManager.PREFERENCES.blockly.colorValue.get() / 100f);

```

Comment:
Is this indentation normal? It seems quite big

Reply:
It's two tabs, which is standard for new lines inside method calls

Reply:
So it's GH being weird